sigh/lib/assets/resign.sh

```diff
@@ -712,48 +785,36 @@ function resign {
                 log "Replacing old app identifier prefix '$OLD_APP_ID' with new value '$NEW_APP_ID'"
                 sed -i .bak "s/$OLD_APP_ID/$NEW_APP_ID/g" "$PATCHED_ENTITLEMENTS"
             elif [[ "$ID_TYPE" == "TEAM_ID" ]]; then
-                # Replace new team identifier with new value
+                # Replace old team identifier with new value
                 log "Replacing old team ID '$OLD_TEAM_ID' with new team ID: '$NEW_TEAM_ID'"
                 sed -i .bak "s/$OLD_TEAM_ID/$NEW_TEAM_ID/g" "$PATCHED_ENTITLEMENTS"
             else
                 continue
             fi
         done
 
+        # Add specific iCloud Environment key to patched entitlements
```

Comment:
This looks interesting.
The reason why I opened #8307 was because we had another app using iCloud that had issues with iTC upload after using fastlane resigner.

This extra step looks like part of "patching/merging" as well.
Our main app doesn't use iCloud so I can't verify this fix on my side.